api

```diff
@@ -1081,51 +1081,24 @@ terminal_manage() { # wrapper for the original terminal_manage function to termi
   terminal_manage_multi "$action $app"
 }
 
-terminal_manage_multi() { #function to install/uninstall/update/refresh multiple apps (or filelists) - uses a terminal and refreshes the app list
-  queue="$1" #one or multiple actions and app names (or filelists of the format 'filelist:path/to/file:path/to/another/file')
```

Comment:
Actually this documentation was somewhat correct. In https://github.com/Botspot/pi-apps/pull/2340/commits/149996ce31fa44cf871e6156130f2287ad072549 (part of the "use manage daemon for GUI updates" PR) we changed from files in a "filelist" format to individual files with each their own "update-file" action. We still support that and that action is used to update the non-app script (eg: manage, api, gui, etc) so we should properly document it here now.

Reply:
Not sure if you are suggesting there is current filelist usage in the master branch. I could not find any.

Reply:
there isn't any "filelist" usage, it only existed in the PR where "filelist" was added and then refactored into "update-file". The documentation here just wasn't corrected to remove "filelist" and add "update-file". That is what I am saying needs to be done, we need to document that "update-file" exists and is used.

Reply:
fixed in https://github.com/Botspot/pi-apps/pull/2708/commits/0e8a591732867bff9a775e474bc4eae9c0d94e32